www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -350,14 +351,16 @@ build configuration file:
             "storePassword": "android",
             "alias": "mykey1",
             "password" : "password",
-            "keystoreType": ""
+            "keystoreType": "",
```

Comment:
Too bad we have to touch multiple lines whenever we add a new entry to the end. I guess not so much that we can do right now:(